src/main/java/uk/gov/companieshouse/pscstatementdataapi/controller/PscStatementController.java

```diff
@@ -44,7 +49,11 @@ public ResponseEntity<Void> processPcsStatement(@RequestHeader("x-request-id") S
                                                     @PathVariable("statement_id") String statementId,
                                                     @RequestBody CompanyPscStatement companyPscStatement
     ) throws JsonProcessingException {
-        logger.info(String.format("Processing psc statement data for company number %s and statement_id %s", companyNumber, statementId));
+        DataMapHolder.get()
+                .companyNumber(companyNumber)
+                .statementId(statementId);
+        logger.info(String.format("Processing psc statement data for company number %s and statement_id %s",
```

Comment:
I think it would be better to use logger.infoContext(), also supplying the contextId